adform-htb.js

```diff
@@ -144,9 +218,36 @@ function AdformHtb(configs) {
         var queryObj = {};
         var callbackId = System.generateUniqueId();
 
-        /* Change this to your bidder endpoint.*/
-        var baseUrl = Browser.getProtocol() + '//someAdapterEndpoint.com/bid';
+        var request = [], item, _key, _value, i, j, k;
 
+        var globalParams = [ [ 'adxDomain', 'adx.adform.net' ], [ 'fd', 1 ], [ 'url', null ], [ 'tid', null ] ];
+        for (i in returnParcels) {
+
+            item = returnParcels[i].xSlotRef;
+            for (j = 0, k = globalParams.length; j < k; j++) {
```

Comment:
This loop is going through the slot config to set values inside globalParams. However, the only key exists in the config is `tid`. Is there a need for an actual loop?